### ISSUE TITLE
fm-mock.d.ts' is not a module.

### ISSUE THREAD

**Issue by jwillinghalpern** [jwillinghalpern/fm-gofer#12]
I'm seeing this in `Problems` in VSCode:
File '/Volumes/Macintosh HD/Users/JoshStorage/Downloads/wv-codes/node_modules/fm-mock/dist/fm-mock.d.ts' is not a module.

**Comment by jwillinghalpern** [comment]
I noticed that adding `export` in front of the mockScript() declaration seems to fix the issue:
`export declare function mockScript(scriptName: string, functionToCall: {`

So figure out how to make the d.ts file include an export.

I noticed that doing this seems to make all the types defined in d.ts auto-completable in a file that imports the package. (That is preferable to none, but we might want to only expose public functions or somehow flag the public ones)

----
Other things I noticed examining svelte's package.json:

1. they have "module": (I wonder if this dynamically returns the module if using `import`)
```json
  "module": "index.mjs",
  "main": "index",
```
2. They have "files":
```json
"files": [
    "types",
    "compiler.*",
    "register.js",
    "index.*",
```
3. They have "types":  `"types": "types/runtime/index.d.ts",`
4. they have these links:
```json
"bugs": {
    "url": "https://github.com/sveltejs/svelte/issues"
  },
  "homepage": "https://github.com/sveltejs/svelte#README",
```
5. They have "exports":
```json
"exports": {
    "./package.json": "./package.json",
    ".": {
      "import": "./index.mjs",
      "require": "./index.js"
    },
```
